Premier league.ipynb

```diff
@@ -0,0 +1,1533 @@
+{
```

Comment:
<p>Niccce</p><p>Please simplify the huge complicated if. it should be a straight forward comparison.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/A4Vision/games-course/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>